.github/workflows/source_aws.yml

```diff
@@ -14,27 +14,17 @@ on:
 
 jobs:
   resolve-runner:
-    timeout-minutes: 5
-    runs-on: ubuntu-latest
-    outputs:
-      runner: ${{ steps.resolve.outputs.runner }}
-    steps:
-      - name: Check if should use large runner
-        id: large-runner
-        # We want to speed runs on the main branch which prime the cache
-        # We allow large runners only in this case to prevent forks from abusing them (it's enforced via runner groups access rules)
-        # IF YOU WANT TO USE A LARGE RUNNER YOU NEED TO ADD THE WORKFLOW TO THE `CloudQuery releases` GROUP IN https://github.com/organizations/cloudquery/settings/actions/runner-groups
-        if: github.event_name == 'push'
-        run: |
-          echo "runner=cloudquery-release-runner" >> $GITHUB_OUTPUT
-      - name: Resolve runner
-        id: resolve
-        run: |
-          RUNNER=${{ steps.large-runner.outputs.runner }}
-          echo "runner=${RUNNER:-"ubuntu-latest"}" >> $GITHUB_OUTPUT
-  plugins-source-aws:
-    timeout-minutes: 45
+    uses: cloudquery/cloudquery/.github/workflows/reusable_resolve_runner.yml@feat/workflows/reusable
+  test:
     name: "plugins/source/aws"
+    needs: [ resolve-runner ]
+    uses: cloudquery/cloudquery/.github/workflows/reusable_test_source.yml@feat/workflows/reusable
+    with:
+      plugin: aws
+      runner: ${{ needs.resolve-runner.outputs.runner }}
+  sanity-check:
```

Comment:
Since the sanity check is now a separate job, we'll need to add it to the wait for all workflow, otherwise a failure of it won't be enforced

Reply:
I thought the failure of the workflow job will trigger failure of the whole workflow, doesn't it?

Reply:
Since it's not a required status check (that is controlled by the wait for all workflow) it will fail, but the PR will still be mergable